extensions/blocks/contact-info/phone/edit.native.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { withPreferredColorScheme } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import save from './save';
+import simpleInput from '../../../shared/simple-input';
+
+import stylesEditor from '../editor.scss';
+import styles from '../style.scss';
+
+const PhoneEdit = props => {
+	const { setAttributes, getStylesFromColorScheme } = props;
+	const placeholderTextColor = getStylesFromColorScheme(
+		styles.placeholder.color,
+		styles.placeholderDark.color
+	);
+	const nativeProps = {
+		keyboardType: 'phone-pad',
+		style: stylesEditor.blockEditorPlainText,
```

Comment:
I see the specified keyboard type, but on my tests (both sim and device) I didn't see any change on the actual keyboard shown. The same for the Email block.

Reply:
Simulator tested: iPhone 11 (13.4)

Reply:
Interesting, are you perchance running on Android? I haven't tested it out on Android yet -- mea culpa.

Reply:
Testing from https://github.com/wordpress-mobile/gutenberg-mobile/pull/2616 solved the problem.
Now I can see the expected keyboard types 🎉